File: ProcessEventType.vb
﻿Namespace kCura.Windows.Process
	''' <summary>
	''' Possible event types for the <see cref="kCura.Windows.Process.ProcessEvent"></see> object.
	''' </summary>
	Public Enum ProcessEventTypeEnum
		''' <summary>
		''' Information message type
		''' </summary>
		Status = 0
		''' <summary>
		''' Warning message type
		''' </summary>
		Warning = 1
		''' <summary>
		''' Error message type
		''' </summary>
		[Error] = 2
	End Enum
End Namespace

File: Source/Relativity.Import.Export/Process/ProcessEventType.cs
// ----------------------------------------------------------------------------
// <copyright file="ProcessEventType.cs" company="Relativity ODA LLC">
//   © Relativity All Rights Reserved.
// </copyright>
// ----------------------------------------------------------------------------

namespace Relativity.Import.Export.Process
{
	/// <summary>
	/// Represents all supported process events that are published by a <see cref="IRunnable"/> process.
	/// </summary>
	public enum ProcessEventType
	{
		/// <summary>
		/// The process event includes a status message.
		/// </summary>
		Status,

		/// <summary>
		/// The process event includes a warning message.
		/// </summary>
		Warning,

		/// <summary>
		/// The process event includes an error message.
		/// </summary>
		Error,
	}
}